charts/portainer/templates/deployment.yaml

```diff
@@ -83,8 +83,26 @@ spec:
               port: 9443
               scheme: HTTPS
               {{- else }}
+                {{- if .Values.enterpriseEdition.enabled }}
+                  {{- if eq (semver .Values.enterpriseEdition.image.tag | (semver "2.7.0").Compare) -1 }}
+              port: 9443
+              scheme: HTTPS
+                  {{- else }}
+              port: 9000
+              scheme: HTTP
+                  {{- end}}
+                {{- else }}
+                  {{- if eq .Values.image.tag "latest" }}
```

Comment:
I remember I tried that before, but there is template parsing error that `.Values.enterpriseEdition.image.tag` format becomes ambiguous for `semver` as "latest" is not "xx.xx.xx" format. 
I will check again to confirm it

Reply:
Just confirmed that if I use `or`, i.e. `{{- if or (eq (semver .Values.image.tag | (semver "2.6.0").Compare) -1) (eq .Values.image.tag "latest") }}`

It will show the error `executing "portainer/templates/deployment.yaml" at <semver .Values.image.tag>: error calling semver: Invalid Semantic Version`